46-lightning-detector/firmware/lightning_detector/lightning_detector.ino

```diff
@@ -0,0 +1,456 @@
+#include "SparkFun_AS3935.h"
+#include <Notecard.h>
+#include "NotecardEnvVarManager.h"
+
+// Uncomment this line and replace com.my-company.my-name:my-project with your
+// ProductUID.
+// #define PRODUCT_UID "com.my-company.my-name:my-project"
+
+#ifndef PRODUCT_UID
+#define PRODUCT_UID ""
+#pragma message "PRODUCT_UID is not defined in this example. Please ensure your Notecard has a product identifier set before running this example or define it in code here. More details at https://bit.ly/product-uid"
+#endif
+
+#ifndef OUTBOUND_MINS
```

Comment:
Probably worth adding a comment to these defines describing what they do, even if it's kind of obvious (to us).